liquibase-couchbase/src/test/java/liquibase/ext/couchbase/mapper/LinesMapperTest.java

```diff
@@ -0,0 +1,64 @@
+package liquibase.ext.couchbase.mapper;
+
+import liquibase.ext.couchbase.provider.DocumentKeyProvider;
+import liquibase.ext.couchbase.provider.factory.DocumentKeyProviderFactory;
+import liquibase.ext.couchbase.types.DataType;
+import liquibase.ext.couchbase.types.Document;
+import liquibase.ext.couchbase.types.File;
+import liquibase.ext.couchbase.types.Value;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoSettings;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@MockitoSettings
+class LinesMapperTest {
+
+    @Mock
+    private DocumentKeyProviderFactory documentKeyProviderFactory;
+    @Mock
+    private DocumentKeyProvider documentKeyProvider;
+    @Mock
+    private File file;
+
+    private final AtomicLong keyHolder = new AtomicLong(1L);
+    private LinesMapper linesMapper;
+
+    @BeforeEach
+    public void setUp() {
+        this.linesMapper = new LinesMapper(documentKeyProviderFactory);
+    }
```

Comment:
Do we need to create this object every time for every test? because it's dependencies are not changed.
I think we can use @InjectMocks for this

Reply:
Done